decidim-system/app/commands/decidim/system/create_api_user.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module Decidim
+  module System
+    class CreateApiUser < Decidim::Command
+      include ::Decidim::System::TokenGenerator
+
+      def initialize(form, current_admin)
+        @form = form
+        @current_admin = current_admin
+      end
+
+      def call
+        return broadcast(:invalid) unless @form.valid?
+
+        transaction do
+          create_api_user
+        end
+
+        broadcast(:ok, @api_user, password_token)
+      end
+
+      private
+
+      attr_reader :form, :current_admin
+
+      def create_api_user
+        @api_user = Decidim.traceability.create!(
+          ::Decidim::Api::ApiUser,
+          current_admin,
+          **api_user_attributes
+        )
+      end
+
+      def api_user_attributes
+        {
+          decidim_organization_id: form.organization,
+          api_key: key_token,
```

Comment:
We should ensure that the `api_key` is unique. There can be collisions with the defined generation strategy.